### PR TITLE
Added documentation for the beat fields

### PR DESCRIPTION
**Pull request by tsg**
This is the Packetbeat part of https://github.com/elastic/libbeat/pull/285

### CODE REVIEW

File: etc/fields.yml [hunk 1]
```diff
@@ -13,9 +13,20 @@ env:
       These fields contain data about the environment in which the
       transaction was captured.
   fields:
-    - name: shipper
+    - name: beat.name
       description: >
-          The name of the Beat that captured the transaction.
+        Name of the Beat sending the log messages. If the shipper name is set
```
Comment (andrewkroh) [inline etc/fields.yml]: "log messages"? Maybe it should say "events" to be generic.

**Comment by andrewkroh** [comment]
We have those [sample outputs](https://github.com/elastic/packetbeat/tree/master/etc/sample_outputs) that are now out of date. I would either update them or remove them.

File: etc/fields.yml (hunk 1 above)
Comment (tsg) [inline etc/fields.yml]: Ah, yes, copypasta from filebeat. I'll fix it.

**Comment by tsg** [comment]
I saw those as well, I'd say we remove them with a separate PR.

**Comment by ruflin** [comment]
@tsg Can you give me write access to your fork. Then I can push the fix directly :-)

**Comment by tsg** [comment]
I forced pushed a fix here.

**Comment by ruflin** [comment]
@tsg ok for you if we merge https://github.com/elastic/packetbeat/pull/367 first as it has more moving parts? We then need to resolve the conflicts between the two.

**Comment by tsg** [comment]
@ruflin yes, sure, it'll be easier to rebase this one.

**Comment by ruflin** [comment]
@tsg Did a force push to your branch.

**Pull request merged**